### DISCUSSION TITLE
Filter "pre-release" and "release"

### DISCUSSION THREAD

**Discussion by mittalyashu** [community/community#6108] [Other Feature Feedback, Questions, & Ideas] [answered]
I was trying to check the next.js [release notes](https://github.com/vercel/next.js/releases) for recent stable release, but got overwhelmed by scrolling past 50 pre-releases 😂

It would be handy to add some kind of filter to preview "release" (stable) or "pre-release".

**Comment by MylesBorins** [comment, marked as answer]
Thanks for pointing this out @mittalyashu, this is definitely not ideal. As this would be a net new feature (the behavior is the same on the old UI) I think we need to hold off on figuring out how to improve this until we clean up edge cases with the new UI. I'll revisit this thread when we have a bit more bandwidth.

**Reply by MylesBorins** [reply]
While we don't have a full fix from this you can now search releases and filter drafts

---

## Search syntax for searching releases in a repository

You can provide text in your search query which will be matched against the title, body, and tag of the repository's releases. You can also combine the following qualifiers to target specific releases.

| Qualifier        | Example
| ------------- | -------------
| `draft:true` | **draft:true** will only match draft releases.
| `draft:false` | **draft:false** will only match published releases.
| <code>tag:<em>TAG</em></code> | **tag:v1** matches a release with the v1 tag and any minor or patch versions within v1, such as v1.0, v1.2, and v1.2.5.
| <code>created:<em>DATE</em></code> | **created:2021** will match releases created from 2021. You can also provide date ranges. For more information, see "[Understanding the search syntax](/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#query-for-dates)."

**Reply by chuckwagoncomputing** [reply]
Does this no longer work?

**Reply by glin7976** [reply]
Same question

**Reply by ldeveber** [reply]
`prerelease:false` works now, but don't be me and put a space in there because that won't work 😂

**Comment by cmj2002** [comment]
Now you can use `prerelease:false`. ([Reference](https://docs.github.com/en/repositories/releasing-projects-on-github/searching-a-repositorys-releases#search-syntax-for-searching-releases-in-a-repository))

**Comment by szamanr** [comment]
this doesn't really help because it's a hidden feature that nobody knows about. it should be a toggle in the UI.

**Comment by Pikuseru** [comment]
Would really love having this on the UI as a dropdown somehow, we've been wondering how to do this ([link](https://github.com/orgs/community/discussions/158014)) until someone found this discussion from 4 years ago. Is it possible to filter using the UI?

**Comment by ayerofieiev-tt** [comment]
@cmj2002 , please move to UI 🙏

**Comment by ville-j-koskela** [comment]
Looks like the tag prefix search only breaks on period. So searching for `tag:v1` matches `v1.0`, `v1.0.0`, etc. 
However, if you have a hyphenated version like `v1.0-rc1` then searching for `tag:v1` or `tag:v1.0` does not find that release.
